wagtail/wagtailadmin/templates/wagtailadmin/shared/search_area.html

```diff
@@ -0,0 +1,6 @@
+{% if not active %}
```

Comment:
Agreed, it is a bit awkward with the disappearing options. I'll see what I can do to differentiate, trying out `icon-larger nolink` on the active link (but still a link). The larger icon does shuffle the list around a bit though.

I'm not sure about renaming it and how that affects translations. The few languages I checked have translations for "other searches" specifically.